spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/KerasLayer.scala

```diff
@@ -183,6 +252,15 @@ abstract class KerasLayer[A <: Activity: ClassTag, B <: Activity: ClassTag, T: C
   }
  // scalastyle:on
 
+  override def parameters(): (Array[Tensor[T]], Array[Tensor[T]]) = {
+    if (this.modules.length > 1) {
+      super.parameters()
```

Comment:
what is `super`? Why not `labor`?

Reply:
Removed if condition. super.parameters() is Container's parameters(). It is redundant here.